crates/pallet-subspace/src/lib.rs

```diff
@@ -135,8 +135,8 @@ struct VoteVerificationData {
     salt: Salt,
     record_size: u32,
     recorded_history_segment_size: u32,
-    max_plot_size: u64,
-    total_pieces: u64,
+    max_plot_size: NPieces,
```

Comment:
Hm, I was thinking to convert all `max_plot_size` instances everywhere to be in bytes instead for consistency that is why in one of the recent PRs and changed things to be that way on the farmer. We really don't measure other "sizes" in pieces elsewhere.